### PR TITLE
Update comments with new style

### LINKED ISSUE #493: Update comment style on functions.nf and test main.nf
The style of the comments has been updated on the `rnaseq` and the `viralrecon` DSL2 pipelines  after adding the `EditorConfig` `Actions` see [here](https://github.com/nf-core/rnaseq/pull/629) and [here](https://github.com/nf-core/viralrecon/pull/188), respectively. On this way all the `.{nf,groovy,conf}` files have the comment style shown below:

```bash
//
// My awesome comment here
//
```

Also, `nf-core modules create` command will be updated to be consistent with the comment style above, see [here](https://github.com/nf-core/tools/issues/1076).
All the current comments on `functions.nf` need to be updated and the `main.nf` of each module test should be updated accordingly.

### PR DESCRIPTION
**Pull request by JoseEspinosa**
As explained on this issue[#493 ] comment style has been updated.

## PR checklist

Closes #493 

- [x] This comment contains a description of changes (with reason).

### CODE REVIEW

File: software/bedtools/complement/functions.nf [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 //
 // Extract name of software tool from process name using $task.process
-// return task_process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()
+//
```
Comment (drpatelh) [inline software/bedtools/complement/functions.nf:7]: Ohhh...what happened here?

**Comment by drpatelh** [comment]
https://github.com/nf-core/modules/blob/60e163ee8468571a138e93c9576b9b9218c857a1/software/gatk4/createsequencedictionary/functions.nf#L8

File: software/bedtools/complement/functions.nf (hunk 1 above)
Comment (JoseEspinosa) [reply software/bedtools/complement/functions.nf:7]: Try to do it with replace in code... 👀

**Comment by JoseEspinosa** [comment]
Following @drpatelh comment I close this PR and will start a fresh one.

**Pull request closed without merging**